client/rest/src/plugins/rosetta/CatapultProxy.js

```diff
@@ -25,6 +25,21 @@ const bigIntToHexString = value => value.toString(16).padStart(16, '0').toUpperC
 const isReceiptSourceLessThanEqual = (lhs, rhs) =>
 	lhs.primaryId < rhs.primaryId || (lhs.primaryId === rhs.primaryId && lhs.secondaryId <= rhs.secondaryId);
 
+const parseTimeSpan = str => {
+	const postfixMap = {
+		ms: 1,
+		s: 1000,
+		m: 60 * 1000,
+		h: 60 * 60 * 1000
+	};
+
+	const matchingPostfix = Object.keys(postfixMap).find(postfix => str.endsWith(postfix));
+	if (!matchingPostfix)
+		throw RosettaErrorFactory.INVALID_REQUEST_DATA;
```

Comment:
Should this be an internal error? 🤔

Reply:
yea, will change to `INTERNAL_SERVER_ERROR`